vnext/Shared/OInstance.cpp

```diff
@@ -370,6 +371,10 @@ InstanceImpl::InstanceImpl(
       Microsoft::ReactNative::PackagerConnection::CreateOrReusePackagerConnection(*m_devSettings);
     }
 
+    if (Microsoft::React::GetRuntimeOptionBool("ForceSystemChakra")) {
```

Comment:
For reviewers: Is this an adequate location to set the JSI override conditionally?